lib/buildkit/client.rb

```diff
@@ -160,7 +160,7 @@ def next_page(next_page)
     end
 
     def build_path(uri)
-      "#{uri.path}?#{uri.query}"
+      CGI.unescape("#{uri.path}?#{uri.query}")
```

Comment:
Does the query need to be unescaped as well? That sound weird to me.

Reply:
I've moved the unescape to the point where we read the `next` page from the link header. Let me know what you think. Thanks!